examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -47,6 +47,7 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
     var defaultNonSamplingDayRange = 1500;
     var defaultSamplingDayRange = 1;
     var defaultSamplingSize = 10;
+    var defaultPointmapLimit = 1000000;
```

Comment:
I think this needs to be placed in the conf file.

Reply:
I looked into the logic for loading parameters from application.conf, and found that every parameter in the file have to be passed through a parameter in the head list of the "index.scala.html" file and the list is already too long. So, do you think it's a good idea to put these pointmap specific params inside that list?

Reply:
If we want to let user to change the maximum number points on the map.